video/avi_decoder.h

```diff
@@ -28,13 +28,13 @@
 #include "common/rect.h"
 #include "common/str.h"
 
-#include "video/video_decoder.h"
 #include "audio/mixer.h"
+#include "video/video_decoder.h"
```

Comment:
Why it sorts the includes?

Reply:
To group related includes, not necessarily a bad thing IMO on files that import a lot of headers